CONTRIBUTORS.txt

```diff
@@ -275,3 +275,4 @@ Yury Selivanov
 Yusuke Tsutsumi
 Марк Коренберг
 Семён Марьясин
+Ben Timby
```

Comment:
The file is supposed to keep alphabetical order.
Please move this line to appropriate place (or just run `sort CONTRIBUTORS.txt -o CONTRIBUTORS.txt`).